test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -92,9 +94,47 @@
   [response]
   (-> response :body cheshire/parse-string))
 
-(deftest ^:integration environment-classes-integration-test
+(deftest ^:integration environment-classes-integration-cache-disabled-test
+  (testing "when environment classes cache is disabled for a class request"
+    (bootstrap/with-puppetserver-running-with-config
+     app
+     (-> {:jruby-puppet {:max-active-instances 1}}
+         (bootstrap/load-dev-config-with-overrides)
+         (ks/dissoc-in [:puppetserver
+                        :environment-class-cache-enabled]))
```

Comment:
I might have commented in the wrong place/not fully explained what I was thinking. I think that the issue I'm really commenting on is that it seems weird to have this set to true in `dev/puppet-server.conf.sample` when it is by default `false`. I agree with everything you've said above, but it seems in that case that it should be set to `false` and explicitly set to `true` not only in the `environment-classes-integration-cache-enabled-test` but also in the other tests that rely on this setting being set to true, such as the `class-info-updated-after-cache-flush-during-prior-request` test.